### PR TITLE
Refresh XProf UI, adresses: #12 and closes: #23

### PR DESCRIPTION
**Pull request by ppikula**
Changed:
* new xprof logo and dark background in the navbar
* added footer with the appliscale logo
* graph legend improvements:
    - make it horizontal
    - allow to turn on/off particular graphs by clicking on te series
    name
* graph tooltip

### CODE REVIEW

**Comment by ppikula** [comment]
Preview here:
https://www.dropbox.com/s/9dop7xv5ckfjrwi/Screenshot%202016-12-21%2017.56.20.png?dl=0

File: priv/app/graph_flot.jsx [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
       series: {
-        shadowSize: 0   // Drawing is faster without shadows
+        shadowSize: 0,   // Drawing is faster without shadows
+      },
+      legend: {
+        position: "se",
+        labelFormatter: this.labelFormatter.bind(this),
+        noColumns: 10
+      },
+      grid: { hoverable: true},
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: Formatting.

File: priv/app/graph_flot.jsx [hunk 2]
```diff
@@ -1,24 +1,48 @@
...
+        content: "%s: %y"
       },
       yaxes:[
         {
           min: 0,
           tickFormatter: function (v) {
             return Math.round(v/10.0)/100.0 + " ms";
           },
-          position: "left"
+          position: "left",
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: superfluous comma.

File: priv/app/graph_flot.jsx [hunk 3]
```diff
@@ -32,41 +56,56 @@ export default class FlotGraph  {
     });
   }
 
+  labelFormatter(label, series) {
+    return "<span class='legend-label' id='" + series.id + this.divid.substr(1) +"'>"
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: You can use string interpolation with backticks.

**Pull request merged**